src/sema/namespace.rs

```diff
@@ -376,7 +377,7 @@ impl Namespace {
             .map(|(id, namespace)| (id, namespace.iter().collect()))
             .unwrap();
 
-        let s = self.resolve_namespace(namespace, file_no, None, id, diagnostics)?;
+        let s = self.resolve_namespace(namespace, file_no, contract_no, id, diagnostics)?;
```

Comment:
Not to be pedantic, but just `s` is a bad variable name

Reply:
> Not to be pedantic, but just `s` is a bad variable name

I agree

Reply:
Renamed to symbol